tests/findings-to-issues.test.sh

```
@@ -0,0 +1,183 @@
+#!/usr/bin/env bash
+# tests/findings-to-issues.test.sh — fixture suites for the new agent + command wire-up.
+# 5 core suites + 4 bonus suites.
+set -u
+THIS_DIR="$(cd "$(dirname "$0")" && pwd)"
+REPO_ROOT="$(cd "$THIS_DIR/.." && pwd)"
+FIX="$THIS_DIR/fixtures/findings-to-issues"
+AGENT_MD="$REPO_ROOT/plugins/uberdev/agents/findings-to-issues.md"
+REVIEW_PR_MD="$REPO_ROOT/plugins/uberdev/commands/review-pr.md"
+SIMPLIFY_MD="$REPO_ROOT/plugins/uberdev/commands/simplify.md"
```

Shellcheck (0.11.0)

[warning] SIMPLIFY_MD appears unused. Verify use (or export if used externally).

(SC2034)